### PR TITLE
Tests modification. Increase MFTF tests coverage.

### PR DESCRIPTION
**Pull request by nmalevanec**
### Description (*)

Tests refactoring and modification.
Add new automated scenarios:
1. https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2551843 
2. https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553271 
3. https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/1620169
4. https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2555409
5. https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2426402
6. https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/2487619
7. https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/1695157
8. https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/2486870
9. https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/1695035
10. https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/2486491
11. https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2426549
12. https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2426592
13. https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2427051
14. https://app.hiptest.com/projects/69435/test-plan/folders/735235/scenarios/1439505
15. https://app.hiptest.com/projects/69435/test-plan/folders/735235/scenarios/1440236
16. https://app.hiptest.com/projects/69435/test-plan/folders/735234/scenarios/1440274
17. https://app.hiptest.com/projects/69435/test-plan/folders/735234/scenarios/2583168
18. https://app.hiptest.com/projects/69435/test-plan/folders/735263/scenarios/1619872
19. https://app.hiptest.com/projects/69435/test-plan/folders/735203/scenarios/2601675
20. https://app.hiptest.com/projects/69435/test-plan/folders/735203/scenarios/1677015
21. https://app.hiptest.com/projects/69435/test-plan/folders/735203/scenarios/2601674
22. https://app.hiptest.com/projects/69435/test-plan/folders/419537/scenarios/2186526
23. https://app.hiptest.com/projects/69435/test-plan/folders/607680/scenarios/2079469
24. https://app.hiptest.com/projects/69435/test-plan/folders/607680/scenarios/2079442
25. https://app.hiptest.com/projects/69435/test-plan/folders/735236/scenarios/1694909

### Fixed Issues (if relevant)

1. none

### Manual testing scenarios (*)

1. none

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ishakhsuvarov** [comment]
@magento run all tests

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminBackordersEnablesOnConfigurationPageAndAppliedToGroupedProductCustomStock.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminBackordersEnablesOnConfigurationPageAndAppliedToGroupedProductCustomStock" extends="AdminCreateGroupedProductCustomStock">
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminBackordersEnablesOnConfigurationPageAndAppliedToGroupedProductCustomStock.xml:11]: This does not seem like meaningful inheritance. Would be great to refactor this down the road.

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCancelOrderWithGroupedProductCustomStockTest.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <test name="AdminCancelOrderWithGroupedProductCustomStockTest" extends="AdminCreateOrderWithGroupedProductDefaultStockTest">
+    <test name="AdminCancelOrderWithGroupedProductCustomStockTest">
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCancelOrderWithGroupedProductCustomStockTest.xml:11]: 👍

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoPartialRefundVirtualProductCustomOptionCustomStock.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateCreditMemoPartialRefundVirtualProductCustomOptionCustomStock" extends="AdminCreateCreditMemoTotalRefundVirtualProductCustomOptionDefaultStock">
+        <annotations>
+            <stories value="Virtual Product Custom Stock."/>
+            <title value="Create Partial Refund Credit Memo with Virtual Product with Custom Option on Custom stock."/>
+            <description value="Verify, admin able to create credit memo with partial refund with Virtual product with custom option on custom stock."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2427051"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoPartialRefundVirtualProductCustomOptionCustomStock.xml:16]: This is already linked to `AdminCreateCreditMemoPartialRefundVirtualProductCustomStock`

Please investigate and update links if approproate

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductCustomStock.xml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductCustomStock" extends="AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductDefaultStock">
+        <annotations>
+            <stories value="Grouped Product Custom Stock."/>
+            <title value="Create Credit Memo with Total Refund after Full Invoice and Full Shipment with Grouped product on Custom stock."/>
+            <description value="Verify, admin able to create credit memo with total refund after full invoice and full shipment with grouped product on custom stock."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553502"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductCustomStock.xml:16]: This is already linked to `AdminCreateCreditMemoTotalRefundGroupedProductCustomStock`

Please investigate and fix (if appropriate)

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoicePartialShipmentGroupedProductCustomStock.xml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateCreditMemoTotalRefundFullInvoicePartialShipmentGroupedProductCustomStock" extends="AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductDefaultStock">
+        <annotations>
+            <stories value="Grouped Product Custom Stock."/>
+            <title value="Create Credit Memo with Total Refund after Full Invoice and Partial Shipment with Grouped product on Custom stock."/>
+            <description value="Verify, admin able to create credit memo with total refund after full invoice and partial shipment with grouped product on custom stock."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553867"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoicePartialShipmentGroupedProductCustomStock.xml:16]: `AdminCreateCreditMemoTotalRefundPartialShipmentGroupedProductCustomStock`

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductCustomStock.xml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductCustomStock" extends="AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductDefaultStock">
+        <annotations>
+            <stories value="Virtual Product Custom Stock."/>
+            <title value="Create Total Refund Credit Memo after Full Invoice With Virtual Product on Custom Stock."/>
+            <description value="Verify admin able to create Credit Memo with total refund after full invoice with virtual product custom stock."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/1695035"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductCustomStock.xml:16]: * AdminCreateCreditMemoTotalRefundVirtualProductDefaultStock
* AdminCreateCreditMemoTotalRefundVirtualProductCustomStock

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreatePartialShipmentGroupedProductCustomStock.xml [hunk 7]
```diff
@@ -8,7 +8,7 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <test name="AdminCreatePartialShipmentGroupedProductCustomStock" extends="AdminCreateCreditMemoTotalRefundGroupedProductDefaultStock">
+    <test name="AdminCreatePartialShipmentGroupedProductCustomStock" extends="additionalStockAdminCreateCreditMemoTotalRefundFullInvoiceGroupedProductWithDefaultStock">
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreatePartialShipmentGroupedProductCustomStock.xml:11]: Is name capitalization part of convention?

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/LoggedInCustomerOrderGroupedProductCustomStock.xml [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="LoggedInCustomerOrderGroupedProductCustomStock" extends="AdminCreateGroupedProductCustomStock">
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/LoggedInCustomerOrderGroupedProductCustomStock.xml:11]: Is it optimal inheritance strategy? Scenarios seem absolutely different

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/LoggedInCustomerReorderVirtualProductOnDefaultStockTest.xml [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+        <waitForPageLoad stepKey="waitUntilOrderPlaced"/>
+        <grabTextFrom selector="{{CheckoutSuccessMainSection.orderNumber22}}" stepKey="orderNumber"/>
+        <see selector="{{CheckoutSuccessMainSection.success}}" userInput="Your order number is:" stepKey="checkOrderPlaceSuccessMessage"/>
+        <!--Reorder simple product.-->
+        <amOnPage url="{{StorefrontCustomerDashboardPage.url}}" stepKey="goToCustomerDashboardPage"/>
+        <waitForPageLoad stepKey="waitForCustomerDashboardPageLoad"/>
+        <actionGroup ref="StorefrontCustomerGoToSidebarMenu" stepKey="goToMyOrdersPage">
+            <argument name="menu" value="My Orders"/>
+        </actionGroup>
+        <click selector="//td[text()='{$orderNumber}']/following-sibling::td[@class='col actions']/a[contains(@class, 'order')]" stepKey="clickReorderBtn"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/LoggedInCustomerReorderVirtualProductOnDefaultStockTest.xml:72]: Why not make action group of reorder?

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/RegisteredCustomerCreateOrderWithSimpleProductAdditionalStockAdditionalWebsiteTest.xml [hunk 10]
```diff
@@ -20,13 +20,10 @@
...
 
         <!--Register customer.-->
-        <click selector="{{StorefrontPanelHeaderSection.createAnAccountLink}}" stepKey="clickOnCreateAccountLink"/>
-        <fillField userInput="{{Simple_US_Customer.firstname}}" selector="{{StorefrontCustomerCreateFormSection.firstnameField}}" stepKey="fillFirstName"/>
-        <fillField userInput="{{Simple_US_Customer.lastname}}" selector="{{StorefrontCustomerCreateFormSection.lastnameField}}" stepKey="fillLastName"/>
-        <fillField userInput="{{Simple_US_Customer.email}}" selector="{{StorefrontCustomerCreateFormSection.emailField}}" stepKey="fillEmail"/>
-        <fillField userInput="{{Simple_US_Customer.password}}" selector="{{StorefrontCustomerCreateFormSection.passwordField}}" stepKey="fillPassword"/>
-        <fillField userInput="{{Simple_US_Customer.password}}" selector="{{StorefrontCustomerCreateFormSection.confirmPasswordField}}" stepKey="fillConfirmPassword"/>
-        <click selector="{{StorefrontCustomerCreateFormSection.createAccountButton}}" stepKey="clickCreateAccountButton"/>
+        <actionGroup ref="RegisterCustomOnStorefrontActionGroup" stepKey="registerCustomer">
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/RegisteredCustomerCreateOrderWithSimpleProductAdditionalStockAdditionalWebsiteTest.xml:23]: 👍

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+            <stories value="Configurable Product Default Stock."/>
+            <title value="Reorder Button Visibility with Configurable Product on Default Stock."/>
+            <description value="Verify,reorder button is not visible after shipment created for order with configurable product on default source."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735208/scenarios/2630678"/>
+            <severity value="CRITICAL"/>
+            <group value="msi"/>
+            <group value="multi_mode"/>
+        </annotations>
+
+        <before>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml:22]: Wouldn't it be easier to create required customer & order with API call and then just check button visibility?

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="ReorderButtonConfigurableProductDefaultStock">
+        <annotations>
+            <stories value="Configurable Product Default Stock."/>
+            <title value="Reorder Button Visibility with Configurable Product on Default Stock."/>
+            <description value="Verify,reorder button is not visible after shipment created for order with configurable product on default source."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735208/scenarios/2630678"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml:16]: * StorefrontButtonReorderForTheOrderWithConfigurableProductOnDefaultStockInCustomerCabinetOnHomepageIsNotPresentIfProductSoldOutTest

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductTestStock.xml [hunk 13]
```diff
@@ -8,23 +8,41 @@
...
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
-    <test name="ReorderButtonConfigurableProductTestStock" extends="AdminCacheValidationConfigurableProductSoldOutCustomStocks">
+    <test name="ReorderButtonConfigurableProductTestStock" extends="ReorderButtonConfigurableProductDefaultStock">
         <annotations>
             <stories value="Configurable Product Custom Stock."/>
             <title value="Reorder Button Visibility with Configurable Product on Custom Stock."/>
             <description value="Verify,reorder button is not visible after shipment created for order with configurable product on custom source."/>
-            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735208/scenarios/2630678"/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735208/scenarios/1676780"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductTestStock.xml:16]: `StorefrontButtonReorderForTheOrderWithConfigurableProductOnTestStockInCustomerCabinetOnHomepageIsNotPresentIfProductSoldOutTest`

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/StorefrontButtonReorderForTheOrderWithConfigurableProductOnTestStockInCustomerCabinetOnHomepageIsNotPresentIfProductSoldOutTest.xml [hunk 14]
```diff
@@ -25,6 +25,7 @@
         <before>
             <remove keyForRemoval="additionalStock"/>
             <createData entity="BasicMsiStockWithMainWebsite1" stepKey="additionalStock" after="customer"/>
+            <remove keyForRemoval="assignMainWebsiteToDefaultStock"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/StorefrontButtonReorderForTheOrderWithConfigurableProductOnTestStockInCustomerCabinetOnHomepageIsNotPresentIfProductSoldOutTest.xml:28]: 👍

File: app/code/Magento/InventoryAdvancedCheckout/Test/Mftf/Test/CreateOrderWithConfigurableProductCustomStockCustomWebsiteBySkuFromCustomerAccountTest.xml [hunk 15]
```diff
@@ -96,14 +96,10 @@
...
-        <fillField userInput="{{Simple_US_Customer.firstname}}" selector="{{StorefrontCustomerCreateFormSection.firstnameField}}" stepKey="fillFirstName" after="clickOnCreateAccountLink"/>
-        <fillField userInput="{{Simple_US_Customer.lastname}}" selector="{{StorefrontCustomerCreateFormSection.lastnameField}}" stepKey="fillLastName" after="fillFirstName"/>
-        <fillField userInput="{{Simple_US_Customer.email}}" selector="{{StorefrontCustomerCreateFormSection.emailField}}" stepKey="fillEmail" after="fillLastName"/>
-        <fillField userInput="{{Simple_US_Customer.password}}" selector="{{StorefrontCustomerCreateFormSection.passwordField}}" stepKey="fillPassword"  after="fillEmail"/>
-        <fillField userInput="{{Simple_US_Customer.password}}" selector="{{StorefrontCustomerCreateFormSection.confirmPasswordField}}" stepKey="fillConfirmPassword" after="fillPassword"/>
-        <click selector="{{StorefrontCustomerCreateFormSection.createAccountButton}}" stepKey="clickCreateAccountButton" after="fillConfirmPassword"/>
+        <actionGroup ref="RegisterCustomOnStorefrontActionGroup" stepKey="registerCustomer">
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdvancedCheckout/Test/Mftf/Test/CreateOrderWithConfigurableProductCustomStockCustomWebsiteBySkuFromCustomerAccountTest.xml:99]: 👍

File: app/code/Magento/InventoryRequisitionList/Test/Mftf/Test/OrderWithMultipleConfigurableProductsAdditionalStock.xml [hunk 16]
```diff
@@ -154,14 +154,10 @@
...
-        <fillField userInput="{{Simple_US_Customer.firstname}}" selector="{{StorefrontCustomerCreateFormSection.firstnameField}}" stepKey="fillFirstName" after="clickOnCreateAccountLink"/>
-        <fillField userInput="{{Simple_US_Customer.lastname}}" selector="{{StorefrontCustomerCreateFormSection.lastnameField}}" stepKey="fillLastName" after="fillFirstName"/>
-        <fillField userInput="{{Simple_US_Customer.email}}" selector="{{StorefrontCustomerCreateFormSection.emailField}}" stepKey="fillEmail" after="fillLastName"/>
-        <fillField userInput="{{Simple_US_Customer.password}}" selector="{{StorefrontCustomerCreateFormSection.passwordField}}" stepKey="fillPassword"  after="fillEmail"/>
-        <fillField userInput="{{Simple_US_Customer.password}}" selector="{{StorefrontCustomerCreateFormSection.confirmPasswordField}}" stepKey="fillConfirmPassword" after="fillPassword"/>
-        <click selector="{{StorefrontCustomerCreateFormSection.createAccountButton}}" stepKey="clickCreateAccountButton" after="fillConfirmPassword"/>
+        <actionGroup ref="RegisterCustomOnStorefrontActionGroup" stepKey="registerCustomer">
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryRequisitionList/Test/Mftf/Test/OrderWithMultipleConfigurableProductsAdditionalStock.xml:157]: 👍

**Review by ishakhsuvarov — APPROVED** [review]
Approving for now. 

Please double check comments and test case links retrospectively.

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoPartialRefundVirtualProductCustomOptionCustomStock.xml (hunk 3 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoPartialRefundVirtualProductCustomOptionCustomStock.xml:16]: I've checked on 2.3-developer branch, and links are different:
AdminCreateCreditMemoPartialRefundVirtualProductCustomStock - has this link https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/1695157
and mentioned above test with custom option has this link - https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2427051

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductCustomStock.xml (hunk 4 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceFullShipmentGroupedProductCustomStock.xml:16]: Same as above, both tests have different links
Grouped with simple product - https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553502
Grouped with virtual product - https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/1620169

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoicePartialShipmentGroupedProductCustomStock.xml (hunk 5 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoicePartialShipmentGroupedProductCustomStock.xml:16]: Not able to find AdminCreateCreditMemoTotalRefundPartialShipmentGroupedProductCustomStock on 2.3-develop

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductCustomStock.xml (hunk 6 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoTotalRefundFullInvoiceVirtualProductCustomStock.xml:16]: Same as above, not able to find AdminCreateCreditMemoTotalRefundVirtualProductDefaultStock and AdminCreateCreditMemoTotalRefundVirtualProductCustomStock on 2.3-develop

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminBackordersEnablesOnConfigurationPageAndAppliedToGroupedProductCustomStock.xml (hunk 1 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminBackordersEnablesOnConfigurationPageAndAppliedToGroupedProductCustomStock.xml:11]: Will be fixed in next PR

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreatePartialShipmentGroupedProductCustomStock.xml (hunk 7 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreatePartialShipmentGroupedProductCustomStock.xml:11]: Will be fixed in next PR

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml (hunk 12 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml:16]: Removed

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductTestStock.xml (hunk 13 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductTestStock.xml:16]: Removed

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml (hunk 12 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml:16]: Will be removed in next PR

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductTestStock.xml (hunk 13 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductTestStock.xml:16]: Will be removed in next PR

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/LoggedInCustomerOrderGroupedProductCustomStock.xml (hunk 8 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/LoggedInCustomerOrderGroupedProductCustomStock.xml:11]: Will be refactored in next PR

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreatePartialShipmentGroupedProductCustomStock.xml (hunk 7 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreatePartialShipmentGroupedProductCustomStock.xml:11]: Will be fixed in next PR

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml (hunk 11 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml:22]: Test removed in favor StorefrontButtonReorderForTheOrderWithConfigurableProductOnDefaultStockInCustomerCabinetOnHomepageIsNotPresentIfProductSoldOutTest as noticed below

**Pull request merged**